src/Middleware/LaratrustMiddleware.php

```diff
@@ -46,7 +46,7 @@ protected function unauthorized()
         $handler = Config::get("laratrust.middleware.handlers.{$handling}");
 
         if ($handling == 'abort') {
-            return App::abort($handler['code']);
+            return App::abort($handler['code'], $handler['message']);
```

Comment:
it could be something like `return App::abort($handler['code'], $handler['message'] ?? '');`

Reply:
yup, clean enough for me pushing this up now